lib/process-services-cloud/src/lib/start-task-cloud/components/people-cloud/people-cloud.component.ts

```diff
@@ -44,31 +44,59 @@ export class PeopleCloudComponent implements OnInit {
     static ROLE_ACTIVITI_USER = 'ACTIVITI_USER';
     static ROLE_ACTIVITI_MODELER = 'ACTIVITI_MODELER';
 
+    static MODE_SINGLE = 'single';
+    static MODE_MULTIPLE = 'multiple';
+
     /** Show current user in the list or not. */
     @Input()
     showCurrentUser: boolean = true;
 
+    /** Mode of the user selection (single/multiple) */
+    @Input()
+    mode: string = PeopleCloudComponent.MODE_SINGLE;
+
+    /** Roles of users to be listed */
+    @Input()
+    roles: string[];
+
+    /** Array of users to be pre-selected. Pre-select all users in multi selection mode and only the first user of the array in single selection mode */
+    @Input()
+    preSelectUsers: IdentityUserModel[];
+
     /** Emitted when a user is selected. */
     @Output()
-    selectedUser: EventEmitter<IdentityUserModel> = new EventEmitter<IdentityUserModel>();
+    selectUser: EventEmitter<IdentityUserModel> = new EventEmitter<IdentityUserModel>();
+
+    /** Emitted when a selected user is removed in multi selection mode. */
+    @Output()
+    removeUser: EventEmitter<IdentityUserModel> = new EventEmitter<IdentityUserModel>();
 
     /** Emitted when an error occurs. */
     @Output()
     error: EventEmitter<any> = new EventEmitter<any>();
 
+    @ViewChild('userInput')
+    private userInput: ElementRef<HTMLInputElement>;
+
+    private _users: IdentityUserModel[] = [];
     users$: Observable<IdentityUserModel[]>;
 
+    private _selectedUsers: IdentityUserModel[];
+    private selectedUsers: BehaviorSubject<IdentityUserModel[]>;
+    selectedUsers$: Observable<IdentityUserModel[]>;
+
     searchUser: FormControl = new FormControl();
 
     _subscriptAnimationState: string = 'enter';
 
-    users: IdentityUserModel[] = [];
-
     dataError = false;
 
-    currentUser: IdentityUserModel;
-
-    constructor(private identityUserService: IdentityUserService) { }
+    constructor(private identityUserService: IdentityUserService) {
+        this._selectedUsers = <IdentityUserModel[]> [];
```

Comment:
why we cannot have
`private _selectedUsers: IdentityUserModel[] = []`
and remove the first one
and why do we need to send the selectedUser again with next ?
`this.selectedUsers.next(this._selectedUsers);`